### PR TITLE
Add feature flag for doing-business-with-us URLs

### PR DESCRIPTION
**Pull request by willbarton**
We're moving the "Doing Business with Us" pages from a mixture of hard-coded HTML and sheer to Wagtail. I've flagged the URLs for these pages with a `WAGTAIL_DOING_BUSINESS_WITH_US` feature flag. 

If `WAGTAIL_DOING_BUSINESS_WITH_US` is checked, the "Doing Business with Us" pages will be served from Wagtail and are expected to live under `/about-us/doing-business-with-us`.

## Changes

- Flag `/about-us/doing-business-with-us` and its child URLs to serve them via Wagtail

## Notes

- `flagged_url` is so much nicer than [the previous alternative](https://github.com/cfpb/cfgov-refresh/pull/2728/files#diff-e45111e649c8f0371d5367b7351f1dd8R136), but it would still be nice to do something about that `lambda`. It's necessary with deal with the difference in arguments between the flagged view and the fallback view.

## Checklist

* [x] PR has an informative and human-readable title
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: cfgov/cfgov/urls.py [hunk 1]
```diff
@@ -12,6 +12,9 @@
 from wagtail.wagtailadmin import urls as wagtailadmin_urls
 from wagtailsharing import urls as wagtailsharing_urls
 
+from wagtail.wagtailcore import views
```
Comment (chosak) [inline cfgov/cfgov/urls.py]: Using the basic Wagtail `views.serve` method will bypass wagtail-sharing logic. Can you try this diff instead?

```diff
diff --git a/cfgov/cfgov/urls.py b/cfgov/cfgov/urls.py
index a3308cf..4e221c9 100644
--- a/cfgov/cfgov/urls.py
+++ b/cfgov/cfgov/urls.py
@@ -11,8 +11,8 @@ from django.shortcuts import render
 from django.views.generic.base import RedirectView, TemplateView
 from wagtail.wagtailadmin import urls as wagtailadmin_urls
 from wagtailsharing import urls as wagtailsharing_urls
+from wagtailsharing.views import ServeView
 
-from wagtail.wagtailcore import views
 from flags.urls import flagged_url
 
 from core.views import ExternalURLNoticeView
@@ -136,16 +136,17 @@ urlpatterns = [
         flagged_url(
             'WAGTAIL_DOING_BUSINESS_WITH_US',
             r'^$',
-            lambda request: views.serve(request, request.path),
+            lambda req: ServeView.as_view()(req, req.path),
             fallback=TemplateView.as_view(
                 template_name='about-us/doing-business-with-us/index.html'),
             name='index'),
         flagged_url(
             'WAGTAIL_DOING_BUSINESS_WITH_US',
             r'^(?P<page_slug>[\w-]+)/$',
-            lambda request, page_slug: views.serve(request, request.path),
+            lambda req, page_slug: ServeView.as_view()(req, req.path),
             fallback=SheerTemplateView.as_view(),
-            name='page')],
+            name='page')
+        ],
         namespace='business')),
 
     url(r'^external-site/$', ExternalURLNoticeView.as_view(),
```

I was able to get this working with wagtail-sharing with both the `about-us/doing-business-with-us/` page as well as a child page at `about-us/doing-business-with-us/child/`.

Comment (willbarton) [inline cfgov/cfgov/urls.py]: Absolutely. 

Also, I had no idea GitHub had `diff` formatting! That's really cool.

**Comment by higs4281** [comment]
When I run this locally, the admin looks for a nonexistent template `flagadmin/index.html`
Am I missing a setup?

**Comment by willbarton** [comment]
@higs4281 reinstall wagtail-flags from the latest version in requirements/base.txt

**Pull request merged**